writing/forgetting.md

```diff
@@ -0,0 +1,32 @@
+# Stop Forgetting, Employ the Machines! 🤖
+
+It's not your fault when you forget to turn up to board game night.
+
+It's not your fault when you absentmindedly went 4 weeks without washing you bedding.
+
+It's not your fault when you promised to get some more dog flea treatment, didn't, and 1 month later are living with an entire mob of fleas, lead by the big one called Klaus.
+
```

Comment:
You're mixing tense with these first 3 "It's not your fault..."  I like present tense best, as you did with the first one.  Make them all present tense.
"It's not your fault when you absentmindedly *go* 4 weeks without washing *your* bedding."
"It's not your fault when you *promise* to get some more dog flea treatment, *don't*, and 1 month later are living with an entire mob of fleas, *led* by the big one you *call* Klaus."